### PR TITLE
[docs] add link to Cheerio API and another example

### PR DESCRIPTION
**Pull request by ChrisWren**
I wasn't clear on what the API was for a wrapper returned from using `render` so I thought this might be helpful.

### CODE REVIEW

File: docs/api/render.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ disservice.
 
 For the purposes of this documentation, we will refer to Cheerio's constructor as
 `CheerioWrapper`, which is to say that it is analogous to our `ReactWrapper` and `ShallowWrapper`
-constructors.
+constructors. [Here](https://github.com/cheeriojs/cheerio#api) is a link to the API available from a `CheerioWrapper` instance.
```
Comment (aweary) [inline docs/api/render.md]: Can we rephrase this to:

> You can reference the [Cheerio API docs](https://github.com/cheeriojs/cheerio#api) for methods available on a `CheerioWrapper` instance.

I'm not a big fan of providing links with "here" if we can integrate it into a natural sentence otherwise

**Comment by ljharb** [comment]
This may need to be updated, since v3 uses cheerio v1.
